### PR TITLE
Fixed Infinispan session cache FAT tests for Windows

### LINKED ISSUE #10041: Fix infinispan session caching FAT test cases to handle Windows URI file format
The server.xml files for successfully completing the infinispan session caching FAT tests need to be updated to handle the MS Windows platform.  Using uri="file:${shared.resource.dir}" fails in Windows as it appears to be using the file name value specified as a host value for a URL and trying to access a remote location on the network instead of looking for a local file.  The fix is to change the httpSessionCache uri value in all the server.xml files (under the publish directory) for the test case servers from:
```
<httpSessionCache uri="file:${shared.resource.dir}/infinispan/infinispan-attr-meta-configuration.xml"...
```
to 
```
<httpSessionCache uri="file:///${shared.resource.dir}/infinispan/infinispan-attr-meta-configuration.xml"...
```

### PR DESCRIPTION
**Pull request by gjwatts**
Modified Infinispan and Hazelcast session cache FAT test server.xml files to include proper formatting of the httpSessionCache uri for the Windows platform.
Pull fixes #10041

### CODE REVIEW

**Comment by gjwatts** [comment]
#build

**Comment by gjwatts** [comment]
#build

**Comment by gjwatts** [comment]
#libbybot

File: dev/com.ibm.ws.session.cache_fat_config/fat/src/com/ibm/ws/session/cache/config/fat/SessionCacheConfigUpdateTest.java [hunk 1]
```diff
@@ -192,11 +201,16 @@ public void testScheduleInvalidation() throws Exception {
...
+        server.setMarkToEndOfLog(); // Only marks messages.log, does not mark the trace file
+        server.setTraceMarkToEndOfDefaultTrace();
         server.updateServerConfiguration(config);
         server.waitForConfigUpdateInLogUsingMark(APP_NAMES, EMPTY_RECYCLE_LIST);
+        String messageToCheckFor = "doScheduledInvalidation scheduled hours are " + Integer.toString(hour1) + " and " + Integer.toString(hour2);
 
-        TimeUnit.SECONDS.sleep(10); // Due to invalidation thread delay
+        // Monitor trace.log and wait for the message "doScheduledInvalidation scheduled hours are X and Y" before the test should continue
+        // This replaces the TimeUnit.SECONDS.sleep(10) used before
+        assertNotNull("Could not find Infinispan message \"" + messageToCheckFor + "\" Has the Infinispan message changed?",
```
Comment (nmittles) [inline dev/com.ibm.ws.session.cache_fat_config/fat/src/com/ibm/ws/session/cache/config/fat/SessionCacheConfigUpdateTest.java]: This is actually not an Infinispan or Hazelcast message, but comes from com.ibm.ws.session.store.common.BackedHashMap.doScheduledInvalidation(). We should update the comment here and in the Infinispan bucket to reflect this. 
`Has the logging in com.ibm.ws.session.store.common.BackedHashMap.doScheduledInvalidation() changed?`

File: dev/com.ibm.ws.session.cache_fat_config_infinispan/fat/src/com/ibm/ws/session/cache/config/fat/infinispan/SessionCacheConfigUpdateTest.java [hunk 2]
```diff
@@ -153,11 +162,16 @@ public void testScheduleInvalidation() throws Exception {
...
+        server.setMarkToEndOfLog(); // Only marks messages.log, does not mark the trace file
+        server.setTraceMarkToEndOfDefaultTrace();
         server.updateServerConfiguration(config);
         server.waitForConfigUpdateInLogUsingMark(APP_NAMES, EMPTY_RECYCLE_LIST);
+        String messageToCheckFor = "doScheduledInvalidation scheduled hours are " + Integer.toString(hour1) + " and " + Integer.toString(hour2);
 
-        TimeUnit.SECONDS.sleep(10); // Due to invalidation thread delay
+        // Monitor trace.log and wait for the message "doScheduledInvalidation scheduled hours are X and Y" before the test should continue
+        // This replaces the TimeUnit.SECONDS.sleep(10) used before
+        assertNotNull("Could not find Infinispan message \"" + messageToCheckFor + "\" Has the Infinispan message changed?",
```
Comment (nmittles) [inline dev/com.ibm.ws.session.cache_fat_config_infinispan/fat/src/com/ibm/ws/session/cache/config/fat/infinispan/SessionCacheConfigUpdateTest.java]: Please see my previous comment and apply here as well.

File: dev/com.ibm.ws.session.cache_fat_config/fat/src/com/ibm/ws/session/cache/config/fat/SessionCacheConfigUpdateTest.java (hunk 1 above)
Comment (gjwatts) [reply dev/com.ibm.ws.session.cache_fat_config/fat/src/com/ibm/ws/session/cache/config/fat/SessionCacheConfigUpdateTest.java]: Thanks for hunting down the specific origin of that message Nate!  I have added that to the code in both places.

File: dev/com.ibm.ws.session.cache_fat_config_infinispan/fat/src/com/ibm/ws/session/cache/config/fat/infinispan/SessionCacheConfigUpdateTest.java (hunk 2 above)
Comment (gjwatts) [reply dev/com.ibm.ws.session.cache_fat_config_infinispan/fat/src/com/ibm/ws/session/cache/config/fat/infinispan/SessionCacheConfigUpdateTest.java]: Ditto!

**Comment by gjwatts** [comment]
#build

**Comment by gjwatts** [comment]
#libbybot

**Pull request merged**